src/test/java/com/google/neighborgood/UserInfoServletTest.java

```diff
@@ -309,14 +307,16 @@ public void emptyInputTest() throws IOException {
     // The entity can't be null
     assertNotNull(entity);
 
+    System.out.println("\n\n\nin test. user id is: " + entity.getKey().getName());
```

Comment:
I think this print statement could be deleted if it is no longer needed.

Reply:
Oh yes, I forgot to remove this, thanks for pointing this out!